ttnn/cpp/pybind11/operations/__init__.hpp

```diff
@@ -83,10 +89,6 @@ void py_module(py::module& module) {
     auto m_complex_unary_backward = module.def_submodule("complex_unary_backward", "complex_unary_backward operations");
     complex_unary_backward::py_module(m_complex_unary_backward);
 
-    auto m_ccl = module.def_submodule("ccl", "collective communication operations");
```

Comment:
Please help preserve history readability hear by moving the original lines back to their original place. We should only see a 1-line diff (addition) for this file